source/includes/versioned-api-snippets/pymongo-client.py

```diff
@@ -0,0 +1,7 @@
+from pymongo.mongo_client import MongoClient
+from pymongo.server_api import ServerApi
+
+// Replace <connection string> with your MongoDB deployment's connection string.
```

Comment:
Comments in Python are preceded with #

Reply:
Bad copy-pasta, thanks for catching this!